test/fixtures.py

```diff
@@ -5,7 +5,8 @@
 from io import StringIO
 from unittest.mock import MagicMock
 
-from test.test_utils import ClientServerPair
+from test.test_utils import ClientServerPair, CALL_TIMEOUT_IN_SECONDS
+import pylsp_jsonrpc
 
 import pytest
```

Comment:
```suggestion

import pytest
import pylsp_jsonrpc
```

This is a third-party import